### PR TITLE
FISH-657 fixed wstx-services deployment errors

### PR DESCRIPTION
**Pull request by lprimak**
## Description
Fixes web services deployment bug (sporradic)

### Blockers
none

### New tests
QuickLook tests pass now,
no exception such as this:
```
ava.lang.IllegalArgumentException: no <deploymentgroup> element
    at com.sun.enterprise.config.serverbeans.Domain$Duck.getDeploymentGroupNamed(Domain.java:672)
    at sun.reflect.GeneratedMethodAccessor88.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.jvnet.hk2.config.Dom.invokeDuckMethod(Dom.java:1129)
    at org.jvnet.hk2.config.Dom.invoke(Dom.java:1080)
    at org.glassfish.config.support.TranslatedConfigView.invoke(TranslatedConfigView.java:208)
    at com.sun.proxy.$Proxy82.getDeploymentGroupNamed(Unknown Source)
    at org.glassfish.ejb.startup.EjbDeployer.event(EjbDeployer.java:472)
    at org.glassfish.kernel.event.EventsImpl.send(EventsImpl.java:131)
    at com.sun.enterprise.v3.server.ApplicationLifecycle.prepare(ApplicationLifecycle.java:562)
```

### Testing Performed
QuickLook tests

### Testing Environment
Mac, JDK 8

### CODE REVIEW

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Review by rdebusscher — APPROVED** [review]
LGTM

File: appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/startup/EjbDeployer.java [hunk 1]
```diff
@@ -469,7 +468,8 @@ public void event(Event event) {
             }
 
             boolean createTimers = true;
-            boolean isDeploymentGroup = domain.getDeploymentGroupNamed(dcp.target) != null;
+            // target could be null for internal apps such as wstx-services or admin console
+            boolean isDeploymentGroup = dcp.target == null ? false : domain.getDeploymentGroupNamed(dcp.target) != null;
```
Comment (svendiedrichsen) [inline appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/startup/EjbDeployer.java:472]: Shouldn't target == null be checked within getDeploymentGroupNamed

Comment (lprimak) [reply appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/startup/EjbDeployer.java:472]: that was about 50/50 decision on my part, so this way "won out" by a hair

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Review by jbee — APPROVED** [review]

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Pull request merged**